### PR TITLE
Add DoubleToIntConverter

### PR DESCRIPTION
**Pull request by bijington**
Documentation added for .NET MAUI Community Toolkit proposals:

- `[Proposal] DoubleToIntConverter ` <https://github.com/CommunityToolkit/Maui/issues/57>

### CODE REVIEW

**Comment by PRMerger4** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.

File: docs/maui/converters/double-to-int-converter.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: .NET MAUI DoubleToIntConverter
+author: bijington
+description: "The DoubleToIntConverter is a converter that allows users to convert an incoming `double` value to an `int` and vice-versa."
```
Comment (jfversluis) [inline docs/maui/converters/double-to-int-converter.md:4]: Where does this description show up? Does it make sense to have backticks for markup in here?

Comment (bijington) [reply docs/maui/converters/double-to-int-converter.md:4]: That's a very good question! I don't know the answer actually. I can remove those as I doubt it works.

**Comment by PRMerger10** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.

File: docs/maui/converters/double-to-int-converter.md (hunk 1 above)
Comment (jfversluis) [reply docs/maui/converters/double-to-int-converter.md]: Yeah I assume it shows up as the summary in search result and such. Also check the other PRs for this, I noticed it with more. You could double-check with @davidbritch

Comment (davidbritch) [reply docs/maui/converters/double-to-int-converter.md]: The Title and Description metadata is used purely for SEO purposes. There's no point using backpacks in any of the metadata (anything between --- and ---).

Comment (jfversluis) [reply docs/maui/converters/double-to-int-converter.md]: Would that be something to include in the tooling to give warnings about that?

Comment (davidbritch) [reply docs/maui/converters/double-to-int-converter.md]: It could be, but a tooling update request succeeding is a rare event.

**Pull request merged**